plugins/tar/receive-app

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+APP="$1"; REV="$2"
+
+# Don't trigger tar build if there is no tarball.
+if [ ! -f "$DOKKU_ROOT/$APP/src.tar" ]; then
+  cat
+else
+  dokku tar-build $APP
```

Comment:
Ditto. Just create a `functions` file that you can source in, similar to how we allow sourcing of `common/functions`.